etc/test-flux.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -eu -o pipefail
+readonly GO=${GO:-go}
+
+log() {
+  local now
+  now=$(date '+%Y/%m/%d %H:%M:%S')
+  echo "[${now}]" "$@"
+}
+
+determine_flux_revision() {
+  local version revision
+  version=$(go list -m -f '{{.Version}}' github.com/influxdata/flux)
```

Comment:
This should have been `"$GO"` like on L31 (I overlooked that in the other PR). Surrouding quotes on the off-chance that the environment variable contains whitespace.